network-store-iidm-impl/src/main/java/com/powsybl/network/store/iidm/impl/ThreeWindingsTransformerImpl.java

```diff
@@ -375,6 +379,16 @@ public void remove() {
         index.notifyRemoval(this);
     }
 
+    public BranchStatus.Status getBranchStatus() {
+        return BranchStatus.Status.valueOf(resource.getAttributes().getBranchStatus());
+    }
+
+    public ThreeWindingsTransformer setBranchStatus(BranchStatus.Status branchStatus) {
+        resource.getAttributes().setBranchStatus(branchStatus.name());
```

Comment:
So we allow null as a valid value? For kind of extension removal. But in that case we end up we a nullpointer exception because of the .name() ?